benchmarks/dpe_bench.cc

```diff
@@ -165,7 +165,7 @@ int main(int argc, char **argv) {
   assert(placed_size == total_placed_size);
 
   // Aggregate placement schemas from the same target
-  if (!result) {
+  if (!result.Succeeded()) {
```

Comment:
I think this should be `result.Succeeded()` (without the `!`), right?

Reply:
Right.